src/materialized/src/bin/materialized/main.rs

```diff
@@ -141,6 +141,20 @@ pub struct Args {
     /// production cluster that happens to be the active Kubernetes context.)
     #[structopt(long, hide = true, default_value = "minikube")]
     kubernetes_context: String,
+    /// The name of this pod
+    #[clap(
+        long,
+        hide = true,
+        env = "POD_NAME",
```

Comment:
The environment variables all start with `MZ_`:

```suggestion
        env = "MZ_POD_NAME",
```

Reply:
is that a convention?